### PR TITLE
Header updates

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #351 :*
https://github.com/bcgov/met-public/issues/351

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

File: met-web/src/components/engagement/banner/BannerWithoutImage.tsx [hunk 1]
```diff
@@ -63,24 +62,13 @@ const BannerWithoutImage = ({ savedEngagement }: BannerProps) => {
...
-                            {isSmallScreen ? (
-                                <MetHeader3 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader3>
-                            ) : (
-                                <MetHeader2 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader2>
-                            )}
+                            <Typography sx={{ fontWeight: 'bold' }} variant="subtitle1" color="black">
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/banner/BannerWithoutImage.tsx:69]: have a feeling this is MetBody. If not can we please add a subtitle component so we can reuse it

File: met-web/src/components/engagement/banner/BannerWithImage.tsx [hunk 2]
```diff
@@ -74,24 +73,13 @@ const BannerWithImage = ({ savedEngagement, children }: BannerProps) => {
...
-                            {isSmallScreen ? (
-                                <MetHeader3 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader3>
-                            ) : (
-                                <MetHeader2 style={{ fontWeight: 600 }} color="black">
-                                    {EngagementDate}
-                                </MetHeader2>
-                            )}
+                            <Typography sx={{ fontWeight: 'bold' }} variant="subtitle1" color="black">
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/banner/BannerWithImage.tsx:80]: let's add a subtitle component so we can reuse it if this is not already there

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Noticed using a new typography to get subtitle1. Suggesting we reuse this if it is not already there. Check MetBody if it is the same

File: met-web/src/components/engagement/banner/BannerWithoutImage.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/banner/BannerWithoutImage.tsx:69]: good idea

**Comment by djnunez-aot** [comment]
Comments have been addressed

**Review by VineetBala-AOT — APPROVED** [review]

**Pull request merged**